readthedocs/projects/models.py

```diff
@@ -697,29 +691,20 @@ def all_active_versions(self):
         """
         return self.versions.filter(active=True)
 
-    def supported_versions(self, flat=True):
+    def supported_versions(self):
         """
         Get the list of supported versions.
         Returns a list of version strings.
         """
         if not self.num_major or not self.num_minor or not self.num_point:
```

Comment:
Maybe we just always let users know there is a newer version if they aren't on the latest, and we only have 1 "supported" version, which is the latest? At least let them know it's not the most up to date.